src/contexts/sales/index.tsx

```diff
@@ -120,29 +120,23 @@ const SaleInfoProvider = ({ children }: Props) => {
     );
   }, [status, at, height, network, saleInfo]);
 
-  const fetchSaleInfo = useCallback(async () => {
+  const fetchSaleInfo = async () => {
     try {
       setStatus(ContextStatus.LOADING);
       if (
         !coretimeApi ||
         coretimeApiState !== ApiState.READY ||
         !relayApi ||
-        relayApiState !== ApiState.READY ||
-        !coretimeApi.query.broker
+        relayApiState !== ApiState.READY
+        //  || !coretimeApi.query.broker
```

Comment:
Let's either remove or uncomment this. I don't really like keeping commented code.

Reply:
Fixed it.